pygitguardian/client.py

```diff
@@ -347,3 +354,37 @@ def quota_overview(
         obj.status_code = resp.status_code
 
         return obj
+
+    # For IaC Scans
+    def iac_directory_scan(
+        self,
+        directory: bytes,
+        scan_parameters: IaCScanParameters,
+        extra_headers: Optional[Dict[str, str]] = None,
+    ) -> Union[Detail, IaCScanResult]:
+
+        result: Union[Detail, IaCScanResult]
+        try:
+            resp = self.request(
+                "post",
```

Comment:
can be simplified a bit, unless I am missing something:

```suggestion
            resp = self.post(
```